conans/test/functional/toolchains/meson/test_cross_compilation.py

```diff
@@ -56,19 +56,18 @@ def build(self):
 
 
 @pytest.mark.tool("meson")
-@pytest.mark.skipif(sys.version_info.major == 2, reason="Meson not supported in Py2")
 @pytest.mark.skipif(platform.system() != "Darwin", reason="requires Xcode")
-@pytest.mark.parametrize("arch, os_, os_version, os_sdk", [
-    ('armv8', 'iOS', '10.0', 'iphoneos'),
-    ('armv7', 'iOS', '10.0', 'iphoneos'),
-    ('x86', 'iOS', '10.0', 'iphonesimulator'),
-    ('x86_64', 'iOS', '10.0', 'iphonesimulator'),
-    ('armv8', 'Macos', None, None)  # MacOS M1
+@pytest.mark.parametrize("arch, os_, os_version, os_sdk, platform_assertion", [
+    ('armv8', 'watchOS', '10.1', 'watchos', "platform 4"),
+    ('armv8', 'iOS', '17.1', 'iphoneos', "platform 2"),
+    ('armv7', 'iOS', '10.0', 'iphoneos', "cmd LC_VERSION_MIN_IPHONEOS"),  # "platform" is missing!
+    ('x86', 'iOS', '10.0', 'iphonesimulator', "cmd LC_VERSION_MIN_IPHONEOS"),  # "platform" is missing!
+    ('x86_64', 'iOS', '10.0', 'iphonesimulator', "cmd LC_VERSION_MIN_IPHONEOS"),  # "platform" is missing!
+    ('armv8', 'Macos', None, None, "platform 1"),  # MacOS M1
+    ('armv8', 'Macos', '10.11', None, "platform 1"),  # MacOS M1
```

Comment:
Those platform checks: https://stackoverflow.com/questions/44188023/how-to-check-a-lib-static-or-dynamic-is-built-for-ios-simulator-or-mac-osx

Reply:
Don't be afraid to put links like that in the test itself, so it is easier for developers to read it without having to blame + track back to Github PR + find this specific comment in the review that might be hidden if we resolve the conversation.